package.json

```diff
@@ -133,7 +133,7 @@
     "madge:circular": "node_modules/.bin/madge ./src --circular"
   },
   "engines": {
-    "node": ">=12.22.10 <19"
+    "node": ">=14.15.0 <19"
```

Comment:
We should take the latest Node 14 version (14.21.0) to lower the risk of not being able to upgrade dependencies if they require a higher node version. This should then also be changed in the CI workflow where the Node 14 version to test is set.

Reply:
How should this look to exclude node 17?

Reply:
`>=14.21.0 <17 || >=18 <19`

Reply:
Thanks!!